### DISCUSSION TITLE
translation of github

### DISCUSSION THREAD

**Discussion by bartgomez** [community/community#21306] [Other Feature Feedback, Questions, & Ideas]
Hello GitHub,
My name is Emre Algül and i am an bilingual student who wants to earn some pocket money. GitHub is a great site to use and there are significant amount of Turkish users of it yet there are no Turkish translation of the site and i believe that makes so many Turkish users to give up while trying to use this wonderful site. To conclude my offer is Turkish translation of the whole GitHub in exchange of some pocket money. I don't seek any absurd amount of money i just want to contribute my familys economy. I tried to send a ticket but i guess i failed to chose right topic and they sent me to here.

![resim_2022-07-26_092726885](https://user-images.githubusercontent.com/75248415/180938331-77d3c980-cf91-466c-b9d4-a1bda686bf7b.png)

contact : algulemre@outlook.com

**Comment by ghost** [comment]
github officials please add your site to weblate.org project Turkish language is required now I want to translate it voluntarily
@nijel @weblate
